src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -4559,10 +4559,6 @@ pub const CPU_WHICH_JAIL: ::c_int = 5;
 pub const RTF_LLDATA: ::c_int = 0x400;
 pub const RTF_FIXEDMTU: ::c_int = 0x80000;
 
-pub const RTM_VERSION: ::c_int = 5;
```

Comment:
Contradicts the commit message. Also, inconsistent with macOS, DragonFly, NetBSD, OpenBSD.